### DISCUSSION TITLE
AI人工智慧機器人，大量進駐服務業。

### DISCUSSION THREAD

**Discussion by awehao** [1101-GS3073-C/Python-Community-GS3073-C#569] [General]
隨著疫情發展，很多服務業也缺人力，這時機器服務生可以帶給我們許多幫助，裡面也有稍微提到機器人無法真正取代人類的一些內容，值得深思
https://www.youtube.com/watch?v=i5VeeUPkFsI

**Comment by number32984** [comment]
的確可以解決當前遇到的人力問題，但長久之後還是有許多問題須改善

**Comment by Chenszuhong** [comment]
確實可以使用 但我覺得在顧客觀感會有很大的疑慮 因爲是人才會感受到溫暖

**Comment by Raywei0214** [comment]
在人生的歷程中，服務業的出現是必然的。吳運鐸在不經意間這樣說過，不參加變革社會的鬥爭，理想永遠是一種幻影。

**Comment by larry40305** [comment]
現在AI取代人力是趨勢，但失業也是一個值得思考議題

**Comment by Yilin20021016** [comment]
這是以後一定會發生的東西 始終是重複性高的東西

**Comment by HsuChen88** [comment]
或許機器人在服務業甚至比某些服務員還更懂得人性呢

**Comment by bww0401** [comment]
我記得很多吃到飽的火鍋店跟燒烤店都開始引用送餐機器人，我自己體驗的感覺是，其實並不會減少人的溫度，因為有些餐點還是會由服務員來送，但又同時能夠兼顧及時送餐的需求!

**Comment by Aerofe** [comment]
使用AI機器人確實可以降低人力付出的成本，但同時我們也需要考慮服務業人員失業的問題，避免讓社會的失業率逐漸上升

**Comment by alvin1226** [comment]
之前用ETC取代高速公路收費員，也是引起不小的抗議，政府就和遠通電信一起解決大量收費員失業的問題，所以說面對AI的進步，我們也必須想辦法來面對
